ios/FluentUI.Demo/FluentUI.Demo/Demos/PreviewCardDemoController.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import FluentUI
+import UIKit
+import SwiftUI
+
+class PreviewCardDemoController: DemoTableViewController {
+    override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
+        super.init(nibName: nibNameOrNil, bundle: nibBundleOrNil)
+    }
+    required init?(coder: NSCoder) {
+        preconditionFailure("init(coder:) has not been implemented")
+    }
+// MARK: - Constraint Constant Attribute Values
+    private struct Constants {
```

Comment:
i believe we still try to declare private struct in the bottom of the class? https://microsoft.github.io/swift-guide/FileOrganization.html

Reply:
Got it, thanks for the resource